deckhouse-controller/entrypoint.sh

```diff
@@ -34,4 +34,37 @@ EOF
 
 cat ${MODULES_DIR}/values-${bundles_map[$bundle]}.yaml >> ${MODULES_DIR}/values.yaml
 
-exec /sbin/tini -- /usr/bin/deckhouse-controller start
+set +o pipefail
+set +e
+
+PID=0
+EXITCODE=0
+
+for SIG in SIGUSR1 SIGUSR2 SIGINT SIGTERM SIGHUP SIGQUIT; do
+  trap "signal_handler ${SIG}" "${SIG}"
+done
+
+signal_handler() {
+  echo "{\"msg\": \"Catch signal ${1}\"}"
+  case "${1}" in
+  "SIGUSR1" | "SIGUSR2")
+    kill "${PID}"
+    wait "${PID}"
+    run_deckhouse
+    ;;
+  *)
+    kill -"${1}" "${PID}"
+    ;;
+  esac
+}
+
+run_deckhouse() {
+  /usr/bin/deckhouse-controller start &
```

Comment:
tiny is missing here. Deckhous is using it to cleanup zombies left after running hooks written on python and bash.

Reply:
fixed